Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.h

```diff
@@ -46,6 +46,7 @@ bool defaultRequiresUserGestureToLoadVideo();
 bool defaultShowModalDialogEnabled();
 bool defaultWebSQLEnabled();
 bool defaultAllowContentSecurityPolicySourceStarToMatchAnyProtocol();
+bool defaultWriteRichTextDataWhenCopyingOrDragging();
```

Comment:
Is this necessary for WK1?

Reply:
Good catch — I added this on accident and forgot to delete it. Will fix!